kong/db/dao/plugins/go.lua

```diff
@@ -385,7 +385,7 @@ do
 
     if instance_info
       and instance_info.id
-      and instance_info.seq == conf.__seq__
+      and instance_info.seq == instance_info.conf.__seq__
```

Comment:
I think we accidentally reverted recent go fixes. Could you drop the revert commits?